nextflow.config

```diff
@@ -26,7 +26,7 @@ params {
     nucleotides_per_second = 1000 // Default interval size
     sentieon = false // Not using Sentieon by default
     skip_qc = null // All QC tools are used
-    target_bed = false // No default TargetBED file for targeted sequencing
+    target_bed = null // No default TargetBED file for targeted sequencing
```

Comment:
there is an open PR currently discussing using null or false, maybe we can wait ehre until that one is merged.